int32[] stops
int32 busid
int32 startnode